custom-plugin-decorators/tag-sorting/tag-sorting.js

```diff
@@ -0,0 +1,11 @@
+const SortTagsAlphabetically = require('./decorator-alpha.js');
```

Comment:
Q -- should we be using Common JS or ESM here? Want to make sure we set the right precedent with the early additions.

Reply:
ESM 👍 Good catch.

Reply:
Sorry, can you explain what you mean here? I'm not actually a JavaScript developer, so I don't understand the question - and while Adam supplied the answer, I am not sure what action is being requested here.